miio/gateway.py

```diff
@@ -151,7 +151,7 @@ def discover_devices(self):
         device_type_mapping = {
             DeviceType.AqaraRelayTwoChannels: AqaraRelayTwoChannels,
             DeviceType.Plug: AqaraPlug,
-            DeviceType.SensorHT: SensorHT,
+            DeviceType.SensorHT: AqaraHT,
```

Comment:
I would keep the old class in place, as maybe the conversion is different or there are other differences.. Better just let it be as it was.

Reply:
Alright I will duplicate those classes again, just a moment and then it is ready to merge

Reply:
done